### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by TheJaredWilcurt** [TheJaredWilcurt/red-perfume#9]
"Red Perfume" lends itself to a lot of good imagery.

Hoping to have a nice and classy logo for the project.

Ideas and prototypes can be submitted to this issue.

**Comment by TheJaredWilcurt** [comment]
Might be cool to have a different bottle of red perfume for each big release

**Comment by TheGiraffe** [comment]
Hi, do you still need a hand with this? I have some design experience and would be interested to help!

**Comment by TheJaredWilcurt** [comment]
@TheGiraffe Yes, absolutely. Hoping to get some prototypes or drafts to give feedback on

**Comment by ngeshlew** [comment]
@TheJaredWilcurt I'm a designer. I can take a stab at it if you are interested. Please let me know.

**Comment by TheJaredWilcurt** [comment]
@ngeshlew yeah, go for it. some thumbnails or prototypes would be good so we could more rapidly go through feedback and pick a direction

**Comment by ngeshlew** [comment]
@TheJaredWilcurt what's the perfume that you are looking at for this release?

**Comment by TheJaredWilcurt** [comment]
I'm pretty open on branding for the project at the moment. currently focused on wrestling with the larger/more complex aspects of the code features.

**Comment by ngeshlew** [comment]
If you can assign me this issue @TheJaredWilcurt , that would be great.

**Comment by ngeshlew** [comment]
@TheJaredWilcurt let me know your thoughts on this 👇🏽

![red-perfume-logo](https://user-images.githubusercontent.com/4710827/105061845-05d91980-5a8b-11eb-93a3-e286235822e6.jpg)

**Comment by TheJaredWilcurt** [comment]
That's fantastic! can you zip and attach the source files?

**Comment by ngeshlew** [comment]
Yeah, creating a PR right now.

**Comment by TheJaredWilcurt** [comment]
you can attach them to this issue. or you can do a PR against the site repo. I tend to put all image source files in an `assets` folder

* https://github.com/red-perfume/red-perfume.github.io

**Comment by ngeshlew** [comment]
@TheJaredWilcurt I've created the PR on the site repo

**Comment by ngeshlew** [comment]
Hey @TheJaredWilcurt did you manage to review the PR?
